collect_app/src/test/java/org/odk/collect/android/formlists/savedformlist/SavedFormListViewModelTest.kt

```diff
@@ -179,6 +180,26 @@ class SavedFormListViewModelTest {
         assertThat(viewModel.isDeleting.getOrAwaitValue(), equalTo(false))
     }
 
+    @Test
+    fun `deleteForms should return 0 if instances can not be deleted`() {
+        val viewModel = SavedFormListViewModel(scheduler, settings, instancesDataService)
+        whenever(instancesDataService.deleteInstances(any())).thenReturn(false)
+
+        val result = viewModel.deleteForms(longArrayOf(1))
+        scheduler.flush()
+        assertThat(result.value!!.value, equalTo(0))
```

Comment:
You can use `getOrAwaitValue()` here (and in the other test) instead of needing the `!!`.

Reply:
Yes, I can but still will need `!!` as the value the livedata stores is Consumable and I need to unpack it see: https://github.com/getodk/collect/pull/5929/commits/85a69baa31e1a815cb3a2197c1113798219786bd